epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -54,6 +61,31 @@ public void bindChanges(@NonNull ColorHolder holder,
     }
   }
 
+  @Nullable
+  @Override
+  public EpoxyItemAnimation bindChangesAnimated(@NonNull ColorHolder view,
+                                                @NonNull EpoxyModel<?> previouslyBoundModel,
+                                                @Nullable EpoxyItemAnimation interruptedAnimation) {
+
+    int currentColor = ((ColorModel) previouslyBoundModel).color;
+    float currentRotation = 0f;
+
+    if(interruptedAnimation != null) {
+      AnimatorSet previousAnim = (AnimatorSet) ((EpoxyItemAnimation.AnimatorWrapper) interruptedAnimation).wrappedAnimator;
```

Comment:
it still seems kinda redundant to me. First you'll have to check the previous model to see if you need to change something, but then you *still* need to check the viewholder in case the previous animation hasn't finished yet. So why not check just the viewholder in the first case? Or you'll have to save your animations somewhere, in the holder probably, which is even more annoying